.coveralls.yml

```diff
@@ -0,0 +1,2 @@
+service_name: travis-pro
+repo_token: tVZhMlTbg8dcMOfD8szWYSOgXigcwzoGs
```

Comment:
I don't think it is a good idea to have your *repo_token* exposed
In addition, I think you don't need to add a .coveralls.yml here.

Reply:
Done!